src/record.py

```diff
@@ -0,0 +1,44 @@
+"""Record class."""
+from parse import get_parser, get_card
+from extract import read_to_dict
+from standardize import standardizer, strip_payment
+from common import flip_spending_sign
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+class CardRecord(object):
```

Comment:
I got a little confused when I first read this both Viral and I thought it referred to a single record, but it looks like it's a class for all the records. What do you think about a name that implies plurality, like `CardRecords` or anything else?

Reply:
It's a record for a single credit card. What you're referring to would probably be an ExpenseRecord?

Reply:
(the singular is in the card, not the record, does that make sense?) Maybe this should just be called `Card` to avoid confusion with how you're using records elsewhere?

Reply:
got it!

I've been thinking of records as the equivalent of "one row in a csv". `Card` sounds good.